CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.0.9
+
+- #110: Do not set autoAlign for plots to allow specification via query options
```

Comment:
`Do not set autoAlign` maybe add `Do not set autoAlign by default`, to let the users know that there is such an option.

Reply:
Good idea, will fix